pyriemann/classification.py

```diff
@@ -524,8 +523,8 @@ def predict(self, covtest):
         """
         dist = self._predict_distances(covtest)
         neighbors_classes = self.classmeans_[np.argsort(dist)]
-        out, _ = stats.mode(neighbors_classes[:, 0:self.n_neighbors], axis=1)
-        return out.ravel()
+        out = np.unique(neighbors_classes[:, 0:self.n_neighbors], axis=1)[:, 0]
```

Comment:
I doubt this is correct

Reply:
you would need to use return_counts and get the entry in out that has the biggest count

Reply:
You are right. Solution is a little bit more complicated
https://stackoverflow.com/a/12300214

I will try to find the simplest solution for 2D case.